nailed-it/New Group/PolishLibraryViewController.swift

```diff
@@ -198,13 +205,21 @@ class PolishLibraryViewController: UIViewController, UICollectionViewDelegate, U
             let string1 = String(describing: $1.distanceVector)
             return string0 < string1
         }
-        self.colors = sortedColors
-        self.collectionView.reloadData()
-        self.collectionView.scrollToItem(at: IndexPath(row: 0, section: 0),
-                                          at: .top,
-                                          animated: true)
+        
+        var sortedAndFiltered = [PolishColor]()
+        for sortedColor in sortedColors! {
+            if abs(color.distanceVector! - sortedColor.distanceVector!) < 0.4 {
```

Comment:
I think this should just be  `if sortedColor.distanceVector! < 0.4`. the distanceVector for the selected color should always be 0, since it is the same spot.